### PR TITLE
test(surveys): add surveys to stories and snapshots

### PR DESCRIPTION
**Pull request by liyiy**
## Problem

surveys to stories

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/surveys/Surveys.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    decorators: [
+        mswDecorator({
+            get: {
+                '/api/projects/:team_id/surveys/': toPaginatedResponse([
+                    MOCK_BASIC_SURVEY,
+                    MOCK_SURVEY_WITH_RELEASE_CONS,
+                ]),
+                '/api/projects/:team_id/surveys/0187c279-bcae-0000-34f5-4f121921f005/': MOCK_BASIC_SURVEY,
+                '/api/projects/:team_id/surveys/0187c279-bcae-0000-34f5-4f121921f006/': MOCK_SURVEY_WITH_RELEASE_CONS,
+            },
```
Comment (neilkakkar) [inline frontend/src/scenes/surveys/Surveys.stories.tsx:122]: you'll need to mock the api calls for the survey counts too

**Review by Twixes — COMMENTED** [review]
Awesome to see this! Just looks like the survey page has some errors

File: frontend/__snapshots__/scenes-app-surveys--survey-view.png
Comment (Twixes) [inline frontend/__snapshots__/scenes-app-surveys--survey-view.png:1]: ❗️

**Comment by liyiy** [comment]
Discussed with @neilkakkar offline that we'll come back to fixing the "survey results" page with all the hogql calls 😅 as it's better to have something at all for now!

**Pull request merged**